src/in_process_server.cpp

```diff
@@ -287,7 +288,8 @@ class wlcs::Client::Impl
     {
         Surface surface{client};
 
-        shell_surface = wl_shell_get_shell_surface(shell, surface);
+        wl_shell_surface * shell_surface = wl_shell_get_shell_surface(shell, surface);
```

Comment:
Coding style; the `*` should bind to the type name.

Reply:
ah shoot